### DISCUSSION TITLE
Editors broken around June 3rd - 5th.

### DISCUSSION THREAD

**Discussion by clonefunnels** [GrapesJS/grapesjs#5934] [Q&A]
UPDATE: 
I think I found the issue.

It appears that all files were deleted from the old GrapesJS directories.

https://grapesjs.com/js/grapes.min.js

WHY????

You'd think you would consider people that may be using the files hosted there and not just delete them.  Now I have to change about 100 files with new URL's.  I guess I should have not trusted the main file locations to remain as they are.

**Comment by mosh-tudor** [comment]
0_0

Did you use the script directly from grapesjs.com??? But why?

You should always use a CDN:
- https://cdnjs.com/libraries/grapesjs
- https://www.jsdelivr.com/package/npm/grapesjs
